Maths/Number-Theory/Tribonacci-Numbers/Tribonacci.java

```diff
@@ -0,0 +1,25 @@
+ import java.util.*;
+ class Tribonacci
+ {
+	 
+        static int tribonacci(int n) {
+        int dp[] = {0, 1, 1};
```

Comment:
https://en.wikipedia.org/wiki/Generalizations_of_Fibonacci_numbers#:~:text=The%20first%20few%20tribonacci%20numbers,of%20Species%20by%20Charles%20R.

Reply:
@ahampriyanshu  It returns the sum of the previous three elements. Do you need elements to be printed?